benchmarks/bench_bgmv_moe.py

```
@@ -0,0 +1,356 @@
+            _sorted_token_indices = data["sorted_token_ids"][
+                _sorted_indices[:num_valid]
+            ]
+            _g_input = data["x"][_sorted_token_indices]
+            g_shrink_out.zero_()
+            g_expand_out.zero_()
+            grouped_mm_bf16(_g_input, g_lora_a, g_m_indptr, out=g_shrink_out)
+            grouped_mm_bf16(g_shrink_out, g_lora_b, g_m_indptr, out=g_expand_out)
+
+        gg_full_time = benchmark_fn(gg_full_fn)
+        results["gg_full_us"] = gg_full_time
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)